### DISCUSSION TITLE
Is it possible to upload project (running filament) to Hostinger web hosting.

### DISCUSSION THREAD

**Discussion by Hikarii14** [filamentphp/filament#6769] [Help]
Hello, we've been trying to upload our work to a hosting service (hostinger), and we have yet to see any videos or guides that actually use filament. Is there a specific way? or is it really possible?

**Comment by danharrin** [comment]
Its probably possible, but its never advised to use shared hosting for any Laravel project. You should use something like Laravel Forge for the best experience.
